reference/calicoctl/ipam/configure.md

```diff
@@ -0,0 +1,50 @@
+---
+title: calicoctl ipam
```

Comment:
"calicoctl ipam configure" ?

Reply:
show and release had same title. So I changed all three of them to "calicoctl ipam show" "calicoctl ipam release" "calicoctl ipam configure"